.gitlab-ci.yml

```diff
@@ -39,12 +30,21 @@ variables:
   npm_config_https_proxy: http://cloud-proxy:3128
   DEPLOY_SERVICE_TAG: ${CI_COMMIT_REF_SLUG}-${CI_COMMIT_SHORT_SHA}
 
-workflow:
+.default_rules: &default_rules
   rules:
-    - if: $CI_COMMIT_TAG
-      variables:
-        DEPLOY_SERVICE_TAG: ${CI_COMMIT_BRANCH}-${CI_COMMIT_TAG}
-    - when: always
+    - if: '($CI_PIPELINE_SOURCE != "schedule" || $RUN_UI_TESTS_ONLY != "true") && $CI_COMMIT_TAG =~ "/^$/"'
+
+.services:
+  allow_failure: true
```

Comment:
Same comment:
https://github.com/NCI-GDC/submission-ui/pull/806#discussion_r1450745387

Reply:
This is for deployment and somehow PE always put allow_failure here I think.